pkg/controller/cronjob/utils.go

```diff
@@ -200,10 +198,9 @@ func getJobFromTemplate(sj *batch.CronJob, scheduledTime time.Time) (*batch.Job,
 	return job, nil
 }
 
-func getTimeHash(scheduledTime time.Time) uint32 {
-	timeHasher := adler32.New()
-	hashutil.DeepHashObject(timeHasher, scheduledTime)
-	return timeHasher.Sum32()
+// Return Unix Epoch Time
+func getTimeHash(scheduledTime time.Time) int64 {
+	return scheduledTime.Unix()
```

Comment:
Is using seconds will give us enough protection against non-unique identifier? When I have many different scheduled jobs, is it possible that two of them will be scheduled at the same second? What if time was changed to 1 hour back (winter time), I think it adding more chances for the collision.